string name
string desc
uint8 type
bool range
float64 max_value
float64 min_value
value_value value